fmdtools/define/block.py

```diff
@@ -1048,37 +1046,14 @@ class FxnBlock(Block):
     Superclass class for functions which is a special type of Block\
     with c and a attributes for CompArch and ASGs, as well as a defined method for propagation
     """
-    def __init__(self, name='', flows={}, c=dict(), a=dict(), local=dict(), args_f=dict(), **kwargs):
+    def __init__(self, name='', flows={}, args_f=dict(), **kwargs):
```

Comment:
This is removing too many attributes and will break certain models/workflows that rely on the ASG and CompArch types:
- c and a are instantiated just a bit below, so they need to be kept here
- local needs to be kept (and I think is supposed to call add_local_to_flowdict, which I think is supposed to happen in comms also--so, would make a good issue/bug report)
- The previous docs for parameters is mostly accurate--p,s,r,m,t, and sp are all parameters that get passed as kwargs to Block. Maybe we need a better way of documenting?

Reply:
It looks like your latest commit only has the changes to search.py--make sure this makes it in and I'll merge the PR